regression/tests/lms/test_demo.py

```diff
@@ -26,3 +26,5 @@ def test_login(self):
         """
         self.login_page.visit()
         self.login_page.login(self.DEMO_COURSE_USER, self.DEMO_COURSE_PASSWORD)
+        self.assertEquals(self.login_page.q(css='.wrapper-header-courses .header-courses').text[0].lower(),
```

Comment:
A couple tweaks to this (and you should scan to see if you have similar patterns that I may have missed in previous code reviews)...
* replace `self.assertEquals` with `self.assertEqual`. The "Equals" with the 's' at the end is [deprecated](https://docs.python.org/2.7/library/unittest.html#deprecated-aliases).
* Set the 'Log-in successful' explicitly as the `msg` to make it clearer. Also, the message should reflect what the test should tell you if it fails, not if it's successful. So, something like this: `self.assertEqual(self.login_page.q...etc, 'my courses', msg='User not logged in as expected.')`